src/hooks/useAnimatedHighlightStyle.ts

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import {InteractionManager} from 'react-native';
+import {Easing, interpolate, interpolateColor, useAnimatedStyle, useSharedValue, withDelay, withSequence, withTiming} from 'react-native-reanimated';
+import CONST from '@src/CONST';
+import useTheme from './useTheme';
+
+type Props = {
+    height: number;
+    shouldHighlight: boolean;
+    highlightDuration?: number;
```

Comment:
Can we use `highlightDuration` for triggering a highlight instead of `shouldHighlight`? The default `highlightDuration` with 0 could mean disabling the highlight and a positive value would trigger the highlight. That way, we can avoid using an extra variable `shouldHighlight`.

Reply:
I don't mind having an extra variable if that makes things more clear. In my opinion, triggering it through `highlightDuration` is a little unclear. We'd have to explicitly add comments to explain our intention which I think we should avoid.